Upstream-Status: Inappropriate [wire]

--- a/avahi-daemon/static-hosts.c	2026-09-24 11:24:16.053271752 -0500
+++ b/avahi-daemon/static-hosts.c	2026-09-24 11:24:16.054195947 -0500
@@ -138,7 +138,7 @@
         p = (h->address.proto == AVAHI_PROTO_INET && config->publish_a_on_ipv6) ||
             (h->address.proto == AVAHI_PROTO_INET6 && config->publish_aaaa_on_ipv4) ? AVAHI_PROTO_UNSPEC : h->address.proto;
 
-        if ((err = avahi_server_add_address(avahi_server, h->group, AVAHI_IF_UNSPEC, p, 0, h->host, &h->address)) < 0) {
+        if ((err = avahi_server_add_address(avahi_server, h->group, AVAHI_IF_UNSPEC, p, AVAHI_PUBLISH_NO_REVERSE, h->host, &h->address)) < 0) {
             avahi_log_error ("Static host name %s: avahi_server_add_address failure: %s", h->host, avahi_strerror(err));
             return;
         }
